### ISSUE TITLE
headings do not increase in increments of 1

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#879]
Heading levels should be hierarchical, only increasing by 1. This ensures the order of headings is semantically correct. 
In testing, the repository homepage threw this error, and it was also found on Collection homepages, the Browse Collections pages, and on the Catalog page. 

[See the Deque aXe help page for this error.](https://dequeuniversity.com/rules/axe/4.3/heading-order?application=AxeChrome)

Pages generating heading semantics errors during test: 

1.  https://ldr.lafayette.edu/
2. https://ldr.lafayette.edu/collections/lafayette-newspaper-archive
3. https://ldr.lafayette.edu/collections/historical-photos
4. https://ldr.lafayette.edu/collections
5. https://ldr.lafayette.edu/catalog

Associated elements / source code:

1.  `<h4>Lafayette Digital Repository</h4>`
2.  `<h4>Works (4454)</h4>`
3.  `<h4>Works (1163)</h4>`
4.  `<h3>Alsace-Lorraine Picture Postcard Collection</h3>`
5.  `<h4 class="search-result-title"><a href="/concern/publications/7h149r06p">Stasis in an expanding universe: A recipe for stable mixed-component cosmological eras</a></h4>`

Selectors: 

1.  `["h4"]`
2.  `[".hyc-bl-title.hyc-blacklight:nth-child(5) > h4"]`
3.  `[".hyc-bl-title.hyc-blacklight:nth-child(5) > h4"]`
4.  `[".row:nth-child(4) > .collection-list-item.col-md-4:nth-child(1) > .thumbnail > .caption > h3"]`
5. ` ["#document_7h149r06p > .search-result-wrapper.row > .search-results-title-row > .search-result-title"]`

**Comment by noraegloff** [comment]
Deque aXe tools severity level = moderate
